keyboards/iris/keymaps/broswen/rules.mk

```diff
@@ -0,0 +1,13 @@
+RGBLIGHT_ENABLE = yes
+BACKLIGHT_ENABLE = no
+MOUSEKEY_ENABLE = no
+MIDI_ENABLE = no
+BLUETOOTH_ENABLE = no
+COMMAND_ENABLE = no
+TERMINAL_ENABLE = no
+AUDIO_ENABLE = yes
+
+
+ifndef QUANTUM_DIR
```

Comment:
This block isn't needed.  It's a hold over from the old makefile system and should be removed.